metadata.json

```diff
@@ -31,12 +31,6 @@
         "18.04",
         "20.04"
       ]
-    },
-    {
-      "operatingsystem": "Debian",
-      "operatingsystemrelease": [
-        "9"
```

Comment:
Usually we submit a separate PR for this and include it in the modulesync PR. I'd also like to add current Debian versions, provided they pass of course.

Reply:
ok, i can do a separate PR to remove debian9. Tests for debian 10 and 11 are unfortunately not passing.
Those i would readd in a separate PR, too. Then we have an up-to-date testsuite to cover them. Would this be acceptable?

Reply:
moved the debian 9 removal to a separate PR and opened another PR for adding  debian 10 and 11 that includes this modulesync PR